### PR TITLE
1384999, update real-time chat with SignalR 2 and MVC 5

### PR DESCRIPTION
**Pull request by john-par**
Fixes #1384999

Update for task to use .NET tutorial template.

### CODE REVIEW

File: aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md [hunk 1]
```diff
@@ -1,202 +1,196 @@
...
+    This step creates the *ChatHub.cs* class and adds a set of script files and assembly references that support SignalR to the project.
 
-    ![Create new hub](tutorial-getting-started-with-signalr-and-mvc/_static/image6.png)
-9. Replace the code in the **ChatHub** class with the following code.
+1. Replace the code in the new *ChatHub.cs* class with this code:
 
     [!code-csharp[Main](tutorial-getting-started-with-signalr-and-mvc/samples/sample1.cs)]
-10. Create a new class called Startup.cs. Change the contents of the file to the following.
+
+1. In **Solution Explorer**, right-click the project and select **Ad** > **Class**.
```
Comment (tdykstra) [inline aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md]: missing a d in Add

File: aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md [hunk 2]
```diff
@@ -1,202 +1,196 @@
...
+
+1. Name the new class *Startup* and add it to the project.
+
+1. Replace the code in the *Startup.cs* class with this code:
```
Comment (tdykstra) [inline aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md]: class file

File: aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md [hunk 3]
```diff
@@ -1,202 +1,196 @@
 ---
 uid: signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc
-title: "Tutorial: Getting Started with SignalR 2 and MVC 5 | Microsoft Docs"
+title: "Real-time chat with SignalR 2 and MVC 5 | Microsoft Docs"
```
Comment (tdykstra) [inline aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md]: Per the template, title should have the word Tutorial in it.

File: aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md [hunk 4]
```diff
@@ -1,202 +1,196 @@
...
-To learn more advanced SignalR developments concepts, visit the following sites for SignalR source code and resources :
+To learn more advanced SignalR developments concepts, go to these sites for SignalR source code and resources:
+
+* [SignalR Project](http://signalr.net)
+
+* [SignalR GitHub and Samples](https://github.com/SignalR/SignalR)
+
+* [SignalR Wiki](https://github.com/SignalR/SignalR/wiki)
+
+## Next Steps
```
Comment (tdykstra) [inline aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md]: lowercase s in steps

File: aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md [hunk 5]
```diff
@@ -1,202 +1,196 @@
...
+
+In this tutorial you learned how to:
```
Comment (tdykstra) [inline aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md]: comma after tutorial

change to follow example of changes in the intro version of this list, i.e, "in this tutorial, you:" and put the verb in past tense in each bullet

File: aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md [hunk 6]
```diff
@@ -1,202 +1,196 @@
...
-## Next Steps
+The code starts the connection and then passes it a function to handle the click event on the **Send** button in the Chat page.
 
-You learned that SignalR is a framework for building real-time web applications. You also learned several SignalR development tasks: how to add SignalR to an ASP.NET application, how to create a hub class, and how to send and receive messages from the hub.
+## Additional resources
 
 For a walkthrough on how to deploy the sample SignalR application to Azure, see [Using SignalR with Web Apps in Azure App Service](../deployment/using-signalr-with-azure-web-sites.md). For detailed information about how to deploy a Visual Studio web project to a Windows Azure Web Site, see [Create an ASP.NET web app in Azure App Service](https://azure.microsoft.com/documentation/articles/web-sites-dotnet-get-started/).
 
-To learn more advanced SignalR developments concepts, visit the following sites for SignalR source code and resources :
+To learn more advanced SignalR developments concepts, go to these sites for SignalR source code and resources:
```
Comment (tdykstra) [inline aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md]: no s in development 

or better yet, replace the whole sentence with "For more information, see the following resources:"

File: aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md [hunk 7]
```diff
@@ -1,202 +1,196 @@
...
+# Tutorial: Real-time chat with SignalR 2 and MVC 5
+
+This tutorial shows how to use ASP.NET SignalR 2 to create a real-time chat application. You'll add SignalR to an MVC 5 application and create a chat view to send and display messages.
+
+In this tutorial you learn how to:
+
+> [!div class="checklist"]
+> * Set up the project
+> * Run the sample
+> * Examine the code
```
Comment (tdykstra) [inline aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md]: you don't really "learn how to examine the code," you examine the code.  Could drop "learn how to" from the intro sentence

**Comment by tdykstra** [comment]
>I am assuming that matching samples don't exist because of the age of the article, but if you know of samples you'd like me to link to, let me know.
I doubt that a sample project exists for this tutorial

File: aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md [hunk 8]
```diff
@@ -1,202 +1,196 @@
 ---
 uid: signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc
-title: "Tutorial: Getting Started with SignalR 2 and MVC 5 | Microsoft Docs"
+title: "Real-time chat with SignalR 2 and MVC 5 | Microsoft Docs"
 author: pfletcher
-description: "This tutorial shows how to use ASP.NET SignalR 2 to create a real-time chat application. You will add SignalR to an MVC 5 application and create a chat view..."
+description: "This tutorial shows how to use ASP.NET SignalR 2 to create a real-time chat application. You will add SignalR to an MVC 5 application and create ..."
```
Comment (tdykstra) [inline aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md]: I would drop "and create ..." and put a period after "application".

File: aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md (hunk 3 above)
Comment (john-par) [reply aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md]: @tdykstra, need some clarification on this one.

In [the User Story](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1375128), the 3rd bullet in the **Description** says, "H1 should begin “Tutorial: “ followed by a verb (“Tutorial” not in ToC entry or meta title)." Also, [the tutorial](https://review.docs.microsoft.com/en-us/help/contribute/templates/dotnet/tutorial-template-dotnet?branch=master) doesn't mention using the word "Tutorial" in the title. Has there been a change that I am unaware of? Am I reading the instructions wrong? Sorry, but I just want to make sure that I understand before I proceed.

Comment (tdykstra) [reply aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md]: That requirement is in the global tutorial template, I didn't realize the .NET template doesn't have it. So for now consider "tutorial" in the metadata title to be an option that I'm suggesting for SEO, but not an absolute requirement of the template. I'll follow up to find out if the omission in the .NET template was on purpose.

Comment (john-par) [reply aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md]: Thanks very much for the clarification. I added "Tutorial:" to the title. If you find out that the omission in the .NET template was on purpose, please let me know and I ill revert the change.

**Comment by john-par** [comment]
@tdykstra all of your feedback has been incorporated.

File: aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md (hunk 3 above)
Comment (tdykstra) [reply aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md]: Omitting that note from the .NET tutorial template was an oversight, and it's been updated to specify that "tutorial" should be in the metadata title.

Comment (john-par) [reply aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md]: Thank you for the verification. I will make sure anyone else on my team working on this project knows about the omission.

**Pull request merged**